cmd/vic-machine/create/create.go

```diff
@@ -1610,7 +1610,18 @@ func splitVnetParam(p string) (vnet string, value string, err error) {
 		return
 	}
 
+	// If the supplied vSphere network contains spaces then the user must supply a network alias. Guest info won't receive a name with spaces.
+	if strings.Contains(vnet, " ") && (len(mapped) == 1 || (len(mapped) == 2 && len(mapped[1]) == 0)) {
```

Comment:
Under what condition will `mapped[1] == 0`?  In other words, what is the input string?